WalletWasabi.Daemon/Config.cs

```diff
@@ -127,6 +127,9 @@ public Config(PersistentConfig persistentConfig, string[] cliArgs)
 			[ nameof(CoordinatorIdentifier)] = (
 				"-",
 				GetStringValue("CoordinatorIdentifier", PersistentConfig.CoordinatorIdentifier, cliArgs)),
+			[ nameof(MaxAllowedCoordinationFeeRate)] = (
```

Comment:
It is okay but the `Allowed` word sound unnecessary. What about `MaxCoordinationFeeRate`?